spec/lib/pg_search/multisearch/rebuilder_spec.rb

```diff
@@ -223,6 +223,31 @@ def foo
             expect(record.pg_search_document).to be_present
           end
         end
+
+        context "when only addicional_attributes is set" do
```

Comment:
This spec does not fail when the changes to the implementation are reverted, so this is not a valid spec for the new implementation.

Reply:
Not sure what you mean 

`Failures:

  1) PgSearch::Multisearch::Rebuilder#rebuild when the model does not define .rebuild_pg_search_documents and multisearchable is not conditional when only addicional_attributes is set calls update_pg_search_document on each record
     Failure/Error: expect(record_1.reload.pg_search_document.additional_attribute_column).to eq("Model::#{record_1.id}")

       expected: "Model::1"
            got: nil

       (compared using ==)
     # ./spec/lib/pg_search/multisearch/rebuilder_spec.rb:247:in `block (6 levels) in <top (required)>'
`

Reply:
OK I will try again... I must have done something wrong!